frontend/src/DropdownMenu/DropdownMenu.tsx

```diff
@@ -0,0 +1,49 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+import Menu from "@material-ui/core/Menu";
+import MenuItem from "@material-ui/core/MenuItem";
+import TranslateIcon from "@material-ui/icons/Translate";
+import "./DropdownMenu.css";
+
+function DropdownMenu() {
+  const [anchorEl, setAnchorEl] = React.useState<null | HTMLElement>(null);
+
+  const handleClick = (event: React.MouseEvent<HTMLButtonElement>) => {
+    setAnchorEl(event.currentTarget);
+  };
+
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  return (
+    <div className="DropdownMenu">
+      <Button
+        aria-controls="simple-menu"
+        aria-haspopup="true"
+        color="primary"
+        onClick={handleClick}
+      >
+        More
+      </Button>
+      <Menu
+        id="simple-menu"
+        anchorEl={anchorEl}
+        keepMounted
+        open={Boolean(anchorEl)}
```

Comment:
Use the [double bang](https://medium.com/dev-genius/vs-boolean-cd502fbd6101), `!!anchorEl`